styles.js

```diff
@@ -7,7 +10,19 @@ export default createGlobalStyle`
     box-sizing: border-box;
   }
 
+  :root {
+    --primary-color: #ff4d4d;
+    --secondary-color: #FF3B17;
+    --third-color: #ffc44d;
+    --fifth-color: #4d17c9;
+    --grey-topics: #757575;
+    --font-size-headlines: 1rem;
+    --font-size-details: 0.8rem;
+    --font-weight-light: 300;
+  }
+
   body {
+    font-family: ${roboto.style.fontFamily};
     margin: 0;
```

Comment:
add the following code: 
 width: 100%;
 height: 100%;
 max-width: 420px;
 margin-inline: auto;

to center the application and have a maximum width for a better imagination